### PR TITLE
Turbo version bug fix

### PR DESCRIPTION
**Pull request by karol-bisztyga**

### CODE REVIEW

File: Common/cpp/Tools/Applier.cpp [hunk 1]
```diff
@@ -35,7 +35,10 @@ bool Applier::apply(jsi::Runtime &rt, std::shared_ptr<BaseWorkletModule> module)
     }
     sharedValues.push_back(sv);
   }
-  
+
+  if (this->errorHandler->getError() != nullptr && this->errorHandler->getError()->handled) {
```
Comment (Szymon20000) [inline Common/cpp/Tools/Applier.cpp:39]: Why do we need it here?  It should be enough to check this in NativeProxy.

Comment (karol-bisztyga) [reply Common/cpp/Tools/Applier.cpp]: fixed 3b7b8a91f96d85c44f230db115b2d07c79cbfcd4

Comment (karol-bisztyga) [reply Common/cpp/Tools/Applier.cpp:38]: ok I checked, this code is needed for iOS, I brought it back in 
3815172
without it on iOS `PreventMultipleEventWorklets` fails in `WorkletFailureTest.js`

Comment (Szymon20000) [reply Common/cpp/Tools/Applier.cpp:33]: I think we can move it to the NativeProxy.

File: android/src/main/java/com/swmansion/reanimated/NodesManager.java [hunk 2]
```diff
@@ -78,6 +78,7 @@
   private List<OnAnimationFrame> mFrameCallbacks = new ArrayList<>();
   private ConcurrentLinkedQueue<Event> mEventQueue = new ConcurrentLinkedQueue<>();
   private boolean mWantRunUpdates;
+  private boolean preventOnAnimationFrame = false;
```
Comment (Szymon20000) [inline android/src/main/java/com/swmansion/reanimated/NodesManager.java:81]: If we check this in applier we do we need it here?

Comment (karol-bisztyga) [reply android/src/main/java/com/swmansion/reanimated/NodesManager.java:81]: It is needed here. Checking from Applier has been moved to NativeProxy on iOS, but on Android moving it to NativeProxy does not work as expected

Comment (karol-bisztyga) [reply android/src/main/java/com/swmansion/reanimated/NodesManager.java]: 933088b

**Pull request merged**